pkg/users/account-create-dialog.js

```diff
@@ -88,6 +89,14 @@ function AccountCreateBody({ state, errors, change }) {
                     </FlexItem>
                 </Flex>
             </FormGroup>
+            <FormGroup label={_("User ID")}
+                       helperTextInvalid={errors && errors.uid}
+                       validated={(errors && errors.uid) ? "error" : "default"}
```

Comment:
completely optional: This can be written as `errors?.uid`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining